estate/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    'name': "Real estate",
+    'depends': ['base'],
+    'application': True,
+    'installable': True,
+    'data': [
+        'security/ir.model.access.csv',
+        'views/estate_menus.xml',
+        'views/estate_property_views.xml'
```

Comment:
You probably saw that there is an error with that view when you try to startup odoo (or the in runbot logs)

The problem is that the order of data imports in the manifest file matters. Indeed, the menu item references `estate_property_action`, which does not yet exist at the time of import of the `views/estate_menus.xml`, as it was imported first. You thus need to first create the views file, then the menus file, which in tern will be able to reference the views

```suggestion
        'views/estate_property_views.xml',
        'views/estate_menus.xml',
```